orienteer-users/src/main/java/org/orienteer/users/util/RegistrationError.java

```diff
@@ -0,0 +1,29 @@
+package org.orienteer.users.util;
+
+import org.apache.wicket.model.IModel;
+import org.apache.wicket.model.ResourceModel;
+
+/**
+ * Registration errors
+ */
+public enum RegistrationError {
```

Comment:
The same - why enum?

Reply:
This enum uses for registration errors labels

Reply:
Additionally: I think in future if we will build more complex system this enum can be useful for indicates different errors